javascript/javascript2/week2/main.js

```diff
@@ -25,11 +25,17 @@ function renderProducts(products) {
 
 renderProducts(products);
 
-const inputName = document.querySelector("#nameFilter");
-
-//Filter by product name
-inputName.addEventListener("keyup", () => {
-    showedProduts = products.filter(listing => listing.name.toLowerCase().includes(inputName.value.toLowerCase()));
-    list.innerHTML = "";
-    renderProducts(showedProduts);
-})
\ No newline at end of file
+//Add filter by max price
+const maximumPrice = document.getElementById("maximumPrice");
+
+let userInputPrice = maximumPrice.addEventListener("input", getTheMaximumPrice);
+
```

Comment:
You don't have `id `called `maximumPrice` and you input in `html file` is `type text` means you will not be able to enter a number.